A/ADMIXTURE/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ADMIXTURE"
+version = v"1.3.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://dalexander.github.io/admixture/binaries/admixture_linux-1.3.0.tar.gz", "353e8b170c81f8d95946bf18bc78afda5d6bd32645b2a68658bd6781ff35703c"),
+    ArchiveSource("http://dalexander.github.io/admixture/binaries/admixture_macosx-1.3.0.tar.gz", "20573453d788e33c01a2756bd9200fd146adf8918b9bb4013d307055926331fc")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+echo "Not originally stated by the author, David Alexander." > LICENSE
```

Comment:
That isn't exactly a great license... If we aren't allowed to redistribute the binaries, we can't, full stop